docs/how-to/how-to-setup-swap-devices-in-guest-kernel.md

```diff
@@ -0,0 +1,58 @@
+# Setup swap device in guest kernel
+
+- [Introduction](#introduction)
```

Comment:
Hi @teawater since https://github.com/kata-containers/kata-containers/pull/2023 has been merged, this TOC should be deleted too.

Reply:
Fixed.